.github/ISSUE_TEMPLATE/pull-request.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Pull Request
+about: Add new Code Examples to Catalog
+title: "[NEW EXAMPLE] <<Language>> <<Service>> <<Description of Examples>>"
+labels: ''
+assignees: ''
+
+---
+
+Each example should contain the following metadata. Some of the fields like keyword and service can be used multiple times as needed. For snippet-service, use the AWS Service Namespace as listed in our [General Reference](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-aws-service-namespaces).  If the example uses multiple services, be sure to include each service on a seperate line. 
+
+Use comment denotations appropriate for each langague.
+Example Metadata and License:
+
+```
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").a
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+*/
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[<<FILENAME>> demonstrates how to ...]
+// snippet-service:[<<ADD SHORT SERVICE NAME>>]
+// snippet-sourcesyntax:[<<LANGUAGE>>]
+// snippet-keyword:[<<LANGUAGE>>]
+// snippet-keyword:[<<ADD FULL SERVICE NAME>>]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[<<API OPERATION>>]
+// snippet-sourcetype:[snippet] or snippet-sourcetype:[full-example]
+// snippet-sourcedate:[YYYY-MM-DD]
+// snippet-sourceauthor:[<<YOUR USERNAME>>]
+// snippet-start:[service.language.unique_name.complete]
```

Comment:
Example?

Reply:
Let me know if the comments above the example metadata are not clear.